### PR TITLE
fix typo in polling source

### PR DESCRIPTION
**Pull request by vsinghal13**

### CODE REVIEW

File: website/docs/r/polling_source.html.markdown [hunk 1]
```diff
@@ -76,7 +76,7 @@ resource "sumologic_polling_source" "terraform_cw_metrics" {
     limit_to_namespaces = ["AWS/Route53","AWS/S3","customNamespace"]
   
     dynamic "tag_filters" {
-    for_each = local.filters
+    for_each = local.tagfilters
     content{
```
Comment (mccartney) [inline website/docs/r/polling_source.html.markdown:80]:
```suggestion
    content {
```

**Comment by mccartney** [comment]
Also an extra `}` in line 87 (cannot submit a suggesting comment in Github)

**Comment by vsinghal13** [comment]
> Also an extra `}` in line 87 (cannot submit a suggesting comment in Github)

why do we need an extra `}` there?

**Comment by mccartney** [comment]
> why do we need an extra } there?

I don't think we need it. It seems like we have 1 too many.

**Pull request merged**